### PR TITLE
Fixed #1598 - AOR - Wrong links in exported PDF

### LINKED ISSUE #1598: AOR - Wrong links in exported PDF
Hi, the links generated in pdf are wrong...

Their urls look  like  http://example./com/?action=ajaxui#ajaxUILoc=index.php?module=unsu_Booking&action=DetailView&record=98d46cb2-9c2e-e496-0587-5750332327a1

after this shows you dashboard, but not the record page

link

### PR DESCRIPTION
**Pull request by shogunpol**
AOR - Wrong links in exported PDF - Fixed
## Description

AOR - Wrong links in exported PDF - Fixed
References issue #1598

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: themes/SuiteP/css/print.css [hunk 1]
```diff
@@ -43,4 +43,26 @@
...
     display: none;
+}
+
+.list tr th {
+    color: #534D64;
+}
+
+body {
+    background: #ffffff;
+    background: #ffffff;
```
Comment (Dillon-Brown) [inline themes/SuiteP/css/print.css]: Duplicate

File: modules/AOR_Reports/AOR_Report.php [hunk 2]
```diff
@@ -700,7 +700,7 @@ function build_report_html($offset = -1, $links = true, $group_value = '', $tabl
             foreach($fields as $name => $att){
                 if($att['display']){
                     $html .= "<td class='' valign='top' align='left'>";
-                    if($att['link'] && $links){
+                    if($att['link']){
```
Comment (Dillon-Brown) [inline modules/AOR_Reports/AOR_Report.php]: PSR-2

File: modules/AOR_Reports/AOR_Report.php [hunk 3]
```diff
@@ -715,7 +715,7 @@ function build_report_html($offset = -1, $links = true, $group_value = '', $tabl
                     if($att['total']){
                         $totals[$name][] = $row[$name];
                     }
-                    if($att['link'] && $links) $html .= "</a>";
+                    if($att['link']) $html .= "</a>";
```
Comment (Dillon-Brown) [inline modules/AOR_Reports/AOR_Report.php]: PSR-2

**Comment by Mac-Rae** [comment]
Please rebase this pull request onto hotfix-7.10.x (For 7.10.x Issues) or hotfix (For 7.11.x Issues) as the Master & Hotfix branches have changed purpose and no longer allow direct PRs since submission. 

Please take a look at our [contribution guidelines](https://github.com/salesagility/SuiteCRM/blob/master/.github/CONTRIBUTING.md) for more information.

**Comment by Dillon-Brown** [comment]
Closed due to inactivity.

**Pull request closed without merging**